app/src/pages/inside/logsPage/tabs/attachments/attachments.jsx

```diff
@@ -0,0 +1,70 @@
+// @ts-check
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { Carousel } from 'react-responsive-carousel';
+import { connect } from 'react-redux';
+import { getIcon } from 'common/img/launch/attachments';
+import { attachmentsSelector } from 'controllers/attachments';
+import 'react-responsive-carousel/lib/styles/carousel.css';
+import styles from './attachments.scss';
+
+const cx = classNames.bind(styles);
+
+@connect((state) => ({
+  logItems: attachmentsSelector(state),
+}))
+export default class Attachments extends React.Component {
+  static defaultProps = {
+    // TODO: throw actions instead and
+    onClickItem: (selectedItem) => console.log('onclick', selectedItem),
+  };
+  static propTypes = {
+    onClickItem: PropTypes.func,
+    logItems: PropTypes.array.isRequired,
+  };
+
+  state = {
+    attachments: (this.props.logItems || []).map((attachment) => ({
```

Comment:
`(this.props.logItems || [])` looks strange because `logItems` marked as required in `propTypes`.
If `logItems` may be empty or null better use `defaultProps` for prevent errors.